### PR TITLE
kubevirt: Add CI for vm-console-proxy

### PR DESCRIPTION
**Pull request by akrejcir**
Adds CI for the repository: https://github.com/kubevirt/vm-console-proxy

### CODE REVIEW

File: ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-main.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.19-openshift-4.12
+images:
+- dockerfile_path: Dockerfile
+  to: vm-console-proxy-container
+releases:
+  initial:
+    integration:
+      name: "4.12"
```
Comment (ksimon1) [inline ci-operator/config/kubevirt/vm-console-proxy/kubevirt-vm-console-proxy-main.yaml:12]: can you use 4.13?

**Comment by akrejcir** [comment]
/pj-rehearse pull-ci-kubevirt-vm-console-proxy-main-e2e-functests

**Comment by ksimon1** [comment]
I think the failing test is related to kubevirt version (v0.58 is not working on 4.13). Kubevirt v0.59 should fix it
/lgtm

**Pull request merged**